lumicks/pylake/force_calibration/power_spectrum_calibration.py

```diff
@@ -184,8 +159,14 @@ def fit_power_spectrum(power_spectrum, model, settings=CalibrationSettings()):
         A power spectrum used for calibration
     model : CalibrationModel
         The model to be used for power spectrum calibration.
-    settings : CalibrationSettings
-        Calibration algorithm settings.
+    analytical_fit_range : tuple (f_min, f_max), optional
+        Tuple of two floats, indicating the frequency range to use for the
+        analytical simple Lorentzian fit, used to obtain initial parameter
+        guesses. Default: (1e1, 1e4) [Hz]
+    ftol : float
+        Termination tolerance for the model fit. Default: 1e-7
+    max_function_evals : int
+        Maximum number of function evaluations during the fit. Default: 10000
```

Comment:
AFAIK the default values appear in Sphinx based on the function signature so there's no need to duplicate them in the description. It's best not to manually duplicate this info because of the bug-prone nature of any duplication: the value could be updated in one place but forgotten in the other. Then the docs end up being inconsistent.

Reply:
Good point. I forgot that you also see the signature when you do `?f` or `help(f)`.